unlock-app/src/components/creator/CreatorLockForm.js

```diff
@@ -135,6 +135,8 @@ export class CreatorLockForm extends React.Component {
   }
 
   render() {
+    const { address } = this.props
+    const disableEdit = address.length === 42
```

Comment:
Do you mind clarify what is 42 here and why this matters?

Reply:
the difference between a new lock and a full lock is the new lock uses a `uniqid()` as its address, which is length 18. A standard lock address is 42. It may be better to explicitly pass in a "editing" prop for this.

Reply:
I think this is too brittle and we should not use the address to determine that a lock is `pending` because we recently discovered that we could get the address of the lock even though the transaction was not mined previously.

There is actually a `pending` value on the lock object which you can re-use here?

Reply:
excellent! That is what I was looking for, thanks